gravitee-apim-portal-webui-next/src/app/api/api-details/api-tab-subscriptions/subscriptions-table/subscriptions-table.component.html

```diff
@@ -36,36 +36,38 @@
       }
     </div>
   } @else {
-    <table mat-table [dataSource]="subscriptions" class="api-tab-subscriptions__table">
-      <ng-container matColumnDef="application">
-        <th mat-header-cell *matHeaderCellDef i18n="@@subscriptionTableColumnApplication" class="m3-title-medium">Application</th>
-        <td mat-cell *matCellDef="let element">{{ element.application }}</td>
-      </ng-container>
+    <div [class.table-mobile]="isMobile()">
```

Comment:
We could make an IsMobileDirective if we start using it often. What do you think?